package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "headway-shipwright-token-style",
   "version": "1.0.0",
-  "description": "to be determine",
+  "description": "to be determined",
   "main": "index.js",
   "scripts": {
-    "build-transform": "npx token-transformer tokens/tokens.json tokens/tokens.json",
+    "build-transform": "npx token-transformer tokens/raw/tokens.json tokens/transformed/tokens.json",
     "build-sd": "style-dictionary build",
-    "build-tw": "postcss ./styles.css -o ./tailwind.css",
+    "build-tw": "npm run build-transform && node build.js",
     "build": "npm run build-transform && npm run build-sd"
```

Comment:
We aren't using package.json for the published automation, you may need to merge in the latest from main in order to get the root level `actions.yml` where the actions of the public script are being maintained.

Reply:
Ok, just rebased onto main!